Framework/CurveFitting/src/Functions/CrystalFieldPeaksBase.cpp

```diff
@@ -65,17 +65,17 @@ void setSymmetryC2(API::IFunction &fun) {
   fun.unfixParameter("B40");
   fun.unfixParameter("B60");
   fixx(fun, "B21");
-  free(fun, "B22", real);
+  free(fun, "B22", REAL_PARAM_PART);
   fixx(fun, "B41");
-  free(fun, "B42", cmplx);
+  free(fun, "B42", CMPLX_PARAM_PART);
```

Comment:
Complex = Real + Imaginary, so not at all interchangeable, i.e., :+1: for changing this.